OSSSpeechKit/Classes/OSSSpeech.swift

```diff
@@ -25,6 +25,46 @@ import UIKit
 import AVFoundation
 import Speech
 
+//MARK: AVSpeechSynthesis Delegate
+
+extension OSSSpeech: AVSpeechSynthesizerDelegate {
+  
+  public func speechSynthesizer(
+    _ synthesizer: AVSpeechSynthesizer, didStart utterance: AVSpeechUtterance) {
+    delegate?.speechSynthesiser(didPerform: .start(utterance))
+  }
+  
+  public func speechSynthesizer(
+    _ synthesizer: AVSpeechSynthesizer, didFinish utterance: AVSpeechUtterance) {
+    delegate?.speechSynthesiser(didPerform: .finish(utterance))
+  }
+  
+  public func speechSynthesizer(
+    _ synthesizer: AVSpeechSynthesizer, didPause utterance: AVSpeechUtterance) {
+    delegate?.speechSynthesiser(didPerform: .pause(utterance))
+  }
+  
+  public func speechSynthesizer(
+    _ synthesizer: AVSpeechSynthesizer,
+    didCancel utterance: AVSpeechUtterance) {
+    delegate?.speechSynthesiser(didPerform: .cancel(utterance))
+  }
+  public func speechSynthesizer(
+    _ synthesizer: AVSpeechSynthesizer,
+    willSpeakRangeOfSpeechString characterRange: NSRange, utterance: AVSpeechUtterance) {
+    delegate?.speechSynthesiser(didPerform: .willSpeakRange(characterRange, utterance))
+  }
+}
+
+//A Simplified CallBack For AVSpeechSynthesizerDelegate
```

Comment:
Please supply correct code documentation commenting. See original code doc comments for guidance. This impacts the code coverage checks.

Reply:
Please check latest commit.